integration/firestore/karma.conf.js

```diff
@@ -19,11 +19,7 @@ const karma = require('karma');
 const path = require('path');
 const karmaBase = require('../../config/karma.base');
 
-const files = [
-  `${path.dirname(require.resolve('firebase'))}/firebase.js`,
-  `${path.dirname(require.resolve('firebase'))}/firebase-firestore.js`,
-  './dist/test-harness.js'
-];
+const files = ['./dist/test-harness.js'];
```

Comment:
These files are not necessary (and I think not there), that code is built into `test-harness.js` now.